packages/client/src/rpc/config.ts

```diff
@@ -0,0 +1,84 @@
+import type { MethodConfig } from '../util/rpc.ts'
+
+export const RPCNamespace = {
+  Eth: 'eth', // untrusted
+  Engine: 'engine', // trusted
+} as const
+
+export type RPCNamespace = (typeof RPCNamespace)[keyof typeof RPCNamespace]
+
+export const RPCTransport = {
+  Ws: 'ws',
+  Http: 'http',
+} as const
+
+export type RPCTransport = (typeof RPCTransport)[keyof typeof RPCTransport]
+
+export class RpcConfig {
+  public static readonly DEFAULT_RPC_ADDR = '0.0.0.0'
+  public static readonly DEFAULT_ENGINE_ADDR = '0.0.0.0'
+  public static readonly DEFAULT_WS_ADDR = '0.0.0.0'
+  public static readonly DEFAULT_WS_ENGINE_ADDR = '0.0.0.0'
+  public static readonly DEFAULT_RPC_PORT = 8545
+  public static readonly DEFAULT_ENGINE_PORT = 8551
+  public static readonly DEFAULT_WS_PORT = 0
```

Comment:
the default websocket port is `8546` and other defaults should be likely made consistent with any defaults that already exist in [`getArgs`](https://github.com/ethereumjs/ethereumjs-monorepo/blob/7d60a4f137ca3aa4370569ba277d4ad262d5d2c1/packages/client/bin/utils.ts#L64)